src/Message/PaymentIntents/ConfirmPaymentIntentRequest.php

```diff
@@ -37,6 +37,10 @@ public function getData()
             $data['return_url'] = $this->getReturnUrl();
         }
 
+        if ($this->getPaymentMethod()) {
+            $data['payment_method'] = $this->getPaymentMethod();
+        }
```

Comment:
Please add some explanation why and how to use this new param - either in the description in top of file: https://github.com/thephpleague/omnipay-stripe/blob/f60aed2fdb81eb91ef721c91c5b418f4733e13d5/src/Message/PaymentIntents/ConfirmPaymentIntentRequest.php#L9 
or above this if.
Would be good if you can add link from official Stripe doc about it.